client/context/query.go

```diff
@@ -81,20 +64,19 @@ func (ctx CLIContext) GetAccount(address []byte) (auth.Account, error) {
 	return account, nil
 }
 
-// GetFromAddress returns the from address from the context's name.
-func (ctx CLIContext) GetFromAddress() (sdk.AccAddress, error) {
-	return ctx.fromAddress, nil
-}
-
-// GetFromName returns the key name for the current context.
-func (ctx CLIContext) GetFromName() (string, error) {
-	return ctx.fromName, nil
+// GetAccAndSeqNums fetches the account and sequence number for a given address
+func (ctx CLIContext) FetchAccAndSeqNums(address sdk.Address) (uint64, uint64, error) {
```

Comment:
This combines two functions which each made a database call with every transaction generated. This reduces that by half.